config/routes.rb

```diff
@@ -664,6 +664,7 @@
               resource :favorite, only: [:create, :destroy]
               resources :favorites, only: [:index]
             end
+            get "catalog" => "datasets#catalog"
```

Comment:
This route should go inside the path /api/v1/data in fact should go in the block in line `642`: 

```
collection do
  get :meta 
  get :catalog
end
```